LiveChat/lc3Parsers.js

```diff
@@ -0,0 +1,52 @@
+const parseRichMessage = (event, user) => {
+    const elements = event.elements[0]
```

Comment:
I believe that it shouldn't be named `elements` as it is just a single element.
Additionally, you can just use array deconstruction:
```suggestion
    const [element] = event.elements[0]
```